app/javascript/shared/helpers/localStorage.js

```diff
@@ -21,6 +22,24 @@ export const LocalStorage = {
     window.localStorage.setItem(key + ':ts', Date.now());
   },
 
+  setFlag(store, accountId, key, expiry = 24 * 60 * 60 * 1000) {
+    const storeName = store ? `${store}::${accountId}` : store;
```

Comment:
Is this conditional statement supposed to be based on the accountId?

```suggestion
    const storeName = accountId ? `${store}::${accountId}` : store;
```